openlibrary/macros/LoanStatus.html

```diff
@@ -34,13 +34,13 @@
       </p>
       <p>
         <a href="$borrow_link" title="Borrow from $contributor"
-	   data-userid="$(user_loan['userid'])"
+	   data-userid="$(user_loan['userid'])" id="read_ebook"
```

Comment:
For all these CSS ID's added in this commit, why use `id` everywhere and not `class`, so that the components can be reused, e.g. on the same page?